src/nstime.c

```diff
@@ -152,12 +152,41 @@ nstime_monotonic_impl(void) {
 }
 nstime_monotonic_t *JET_MUTABLE nstime_monotonic = nstime_monotonic_impl;
 
+static void
+nstime_get_realtime(nstime_t *time) {
+  struct timespec ts;
+
+  clock_gettime(CLOCK_REALTIME, &ts);
```

Comment:
I think this needs to be ifdef-guarded; CLOCK_REALTIME isn't guaranteed to be defined.

Reply:
Let me know what you think about the posix comment above and I'll act accordingly.

Reply:
(I think this one still applies, since e.g. windows doesn't have these).

Reply:
I may well be missing something on platform support so please bear with me. I don't see how this code could be called on a Windows platform.

This code can only be called on a platform where 'opt_prof_time_resolution == prof_time_resolution_high' and the opt processing code code in jemalloc.c ensures that this can only be set if the compile time code detected that  clock_gettime(CLOCK_REALTIME) was available on this platform, i.e., config_high_resolution_timer = true: this should never be the case for a Windows compilation environment. 

Like I say, I may be missing something.